api/app_config.py

```diff
@@ -2,28 +2,18 @@
 
 class AppConfig:
     def __init__(self):
-        self._MONGO_USERNAME = os.environ.get('MONGO_USERNAME')
-        self._MONGO_PASSWORD = os.environ.get('MONGO_PASSWORD')
-        self._MONGO_CLUSTER_NAME = os.environ.get('MONGO_CLUSTER_NAME')
-        self._MONGO_DEFAULT_DB_NAME = os.environ.get('MONGO_DB_NAME')
+        self._MONGO_DB_NAME = os.environ.get('MONGO_DB_NAME')
+        self._MONGO_CONNECTION_STRING = os.environ.get('MONGO_CONNECTION_STRING')
```

Comment:
This looks sensible, but doesn't seem to match the version that's currently deployed to Azure (which is still pointed to Trello)